cypress/e2e/CMS/$BlogSeries.feature

```diff
@@ -0,0 +1,196 @@
+            | Meta Description         | Automated Blog Post2 Meta Description             | field_page_description         |
+            | Feature Card Description | Automated Test Blog Post2 - Feature Card Desc     | field_feature_card_description |
+            | Author                   | Automated Test Blog Post2 Author                  | field_author                   |
+        And user enters "Blog Post List Description" into "List Description" text field
+        And user enters "Blog Post Intro Text" as intro text
+        And user fills out "Body" text area with "This is a description of Blog Post content type."
+        And user selects 1 Lead Image from the list of images for blog series
+        And browser waits
+        And user remembers the source of selected lead image for blog series for further verification
+        And user selects 2 Promotional Image from the list of images for blog series
+        And browser waits
+        And user remembers the source of selected promo image for blog series for further verification
+        And user selects "Published" from "Save as" dropdown
+        Then user saves the content page
+
+    Scenario: Verify edited content in the blog series
+        Given user is navigating to the front end site with path site section plus "blog-series-edited"
+        Then page title is "Automated Test Blog Series Edited"
+        And the "Featured Posts" managed list appears
+        And public use text is not displayed
+        And the Featured Posts section is displayed with the following cards
+            | title                                 | linkEnding | expectedDate | author                          |
+            | Automated Test Blog Post - Card Title | /blog-post | today's date | Automated Test Blog Post Author |
+            | Automated Test Blog Post - Card Title | /blog-post | today's date | Automated Test Blog Post Author |
+            | Automated Test Blog Post - Card Title | /blog-post | today's date | Automated Test Blog Post Author |
+        And the page contains meta tags with the following names
+            | name        | content                                            |
+            | description | Automated Test Blog Series Meta Description Edited |
+        Then list of blog post has the following posts
+            | title                                             | url                                                                                         | expectedDate | author                           |
+            | Automated Test Blog Post for testing Blog Series  | {TEST_SITE_SECTION}/blog-series-{RANDOM}/{YEAR}/blog-post-test-blog-series-{RANDOM}         | today's date | Automated Test Blog Post Author  |
+            | Automated Test Blog Post2 for testing Blog Series | {TEST_SITE_SECTION}/blog-series-edited-{RANDOM}/{YEAR}/blog-post2-test-blog-series-{RANDOM} | today's date | Automated Test Blog Post2 Author |
+        And browser waits
```

Comment:
you should not need any extra waits in the front end verification - as soon as page has loaded, unless you are performing some actions on it, you do not need to wait anymore

Reply:
done